pkg/eve/runme.sh

```diff
@@ -111,6 +111,10 @@ do_version() {
   echo /bits/*.squash | sed -e 's#/bits/rootfs-##' -e 's#.squash##' >&3
 }
 
+do_build_config() {
+  unsquashfs -cat /bits/rootfs.img /etc/linuxkit-eve-config.yml
```

Comment:
You should redirect the output as we do it for `do_version`. Without redirection you will not be able to run notation like `cfg=$(docker run --rm docker.io/evebuild/danger:pr2849-kvm build_config)`, because output will be in stderr, not in stdout.

Reply:
`unsquashfs -cat` dumps to stderr? How odd. Ah, no, we do this weird redirect thing at the beginning of `runme.sh`

OK, I will change it.

Reply:
Done